package.json

```diff
@@ -25,6 +25,11 @@
   "author": "devsupport@webex.com",
   "license": "MIT",
   "dependencies": {
+    "@webex/components": "^1.18.1",
+    "prop-types": "^15.7.2",
```

Comment:
I think `prop-types` is not necessary to add.

Reply:
![img](https://media0.giphy.com/media/d3mlE7uhX8KFgEmY/giphy.gif)

It didn't work in node without it, but jest works fine with it. Removing.